app/src/auto_validator/core/models.py

```diff
@@ -22,64 +28,53 @@ def url(self):
 
 
 class Block(models.Model):
-    serial_number = models.IntegerField(primary_key=True, unique=True)  # Unique identifier for the block
-    timestamp = models.DateTimeField()  # Time when the block was created
+    serial_number = models.IntegerField(primary_key=True, unique=True)
+    timestamp = models.DateTimeField()
 
     def __str__(self):
-        return f"Block {self.serial_number} at {self.timestamp}"
+        return f"{self.serial_number}"
 
 
 class Subnet(models.Model):
     name = models.CharField(max_length=255)
-    description = models.TextField(null=True, blank=True)  # Additional field for description
-    operators = models.ManyToManyField("Operator", related_name="subnets", blank=True)  # Link to operators
-    running_mainnet = models.BooleanField(default=False)
-    running_testnet = models.BooleanField(default=False)
+    description = models.TextField(null=True, blank=True)
+    operators = models.ManyToManyField("Operator", related_name="subnets", blank=True)
```

Comment:
will it work well with blank but without null?

Reply:
![image](https://github.com/user-attachments/assets/676a27f3-3857-401e-9357-33f7a8071555)